apps/beets-frontend-v3/lib/modules/lst/hooks/useGetAmountDelegatedPerValidator.tsx

```diff
@@ -65,10 +65,12 @@ export function useGetAmountDelegatedPerValidator(chain: GqlChain) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [stakeRequests, isLoading])
 
-  const amountDelegatedPerValidator = validatorIds.map(validatorId => ({
-    validatorId,
-    amountDelegated: amountResults[validatorId]?.result ?? 0n,
-  }))
+  const amountDelegatedPerValidator = validatorIds
+    .map(validatorId => ({
+      validatorId,
+      amountDelegated: amountResults[validatorId]?.result ?? 0n,
+    }))
+    .filter(validator => ['13', '14', '24', '29', '30'].includes(validator.validatorId))
```

Comment:
would probably add it to some kind of named list so they don't appear as magic numbers.